vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/finder/StepDefinitionFinder.java

```diff
@@ -107,7 +157,7 @@ private List<StepDefinition> findCompositeSteps(List<IPackageFragment> fragments
                         .flatMap(Arrays::stream)
                         .filter(IJarEntryResource.class::isInstance)
                         .map(IJarEntryResource.class::cast)
-                        .filter(e -> e.getName().endsWith(".steps"))
+                        .filter(e -> ExtensionUtils.isCompositeFile(e.getName()))
```

Comment:
```suggestion
                        .filter(e -> ExtensionUtils.isCompositeFile(e.toString()))
```
?

Reply:
toString() for IJarEntryResource returns either "JarEntryDirectory["+ ... +"]" or "JarEntryFile["+ ... +"]"